data/human/news.txt

```diff
@@ -1695,6 +1854,13 @@ news "pirate"
 			"Living on the edge."
 			"You can't win the game if you're afraid to lose it."
 			"Good things come to those who wait, and then I rob those from them very quickly."
+			"Two heads are better than one, since you can put a spiked head on each side of the road if you have two."
+			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to my demands."
```

Comment:
```suggestion
			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to any demands."
```

Reply:
I think this works better with "my demands", because we have a pirate talking here about how he/she is doing business, and "my demands" sounds more self-centered (which is what I expect from someone that kills to get his/her demands met).

Reply:
I suppose, though I'm still not sure if the pirate is _telling_ you.

Reply:
It also works if the pirate is talking to another pirate. The pirate doing the talking is self-centered, regardless of whom he/she is speaking to.